python/tvm_ffi/testing.py

```diff
@@ -16,23 +16,35 @@
 # under the License.
 """Testing utilities."""
 
-from typing import Any, ClassVar
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, ClassVar
 
 from . import _ffi_api
 from .core import Object
 from .dataclasses import c_class, field
 from .registry import register_object
 
+if TYPE_CHECKING:
+    from tvm_ffi import Array, Map
```

Comment:
Given array map is used below (and notunder type chekcing), should it now not use type checking? or use String?